src/customers/CustomersFiltersSidebar.js

```diff
@@ -0,0 +1,17 @@
+import { Card, CardContent } from "@material-ui/core";
+import * as React from "react";
+import { HasNewsletterFilter } from "./CustomerHasNewsletterFilter";
+import { HasOrderedFilter } from "./CustomerHasOreredFilter";
+import { LastVisitedFilter } from "./CustomerLastVisitFilter";
+import { CustomersFilter } from "./CustomersFilter";
+
+export const CustomersFilterSidebar = () => (
+    <Card style={{ order: -1 }}>
+        <CardContent>
+            <CustomersFilter />
```

Comment:
I'm not sure that's what @fzaninotto meant. You're including a Filter component here